src/Core/ModulesApplication.php

```diff
@@ -140,8 +141,10 @@ private function bootstrapCustomModules(ModulesClassLoader $classLoader, $eventD
                 // Can't include a missing bootstrap. Notify user and log the error.
                 error_log("Custom module " . errorLogEscape($customModulePath . $row['mod_directory'])
                     . '/' . self::CUSTOM_MODULE_BOOSTRAP_NAME
-                    . " is enabled but after 3 tries can not read the bootstrap.php script. Uninstall and or disable in module manager.");
+                    . " is enabled but after 3 tries can not read the bootstrap script. System will force disable module.");
                 $failed_modules[] = ["name" => $row["mod_name"], "directory" => $row['mod_directory'], "path" => $customModulePath . $row['mod_directory'], "available" => false, "error" => "Module is missing bootstrap."];
+                // disable
+                sqlStatementNoLog("UPDATE modules SET mod_active = 0 WHERE mod_directory = ? AND type != 1", [$row['mod_directory']]);
```

Comment:
could this statement ever fail?  I don't think so but if it does would we want to log that as well?  Other than that I don't see any problems.